### PR TITLE
LDraw.AIOI version 9.5.0.0

### PR DESCRIPTION
**Pull request by superusercode**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109369&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.InvalidOperationException: No process is associated with this object.

**Comment by stephengillie** [comment]
In a Windows 11 VM, this package didn't complete installing - it started but seemed to pause.

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
+PackageLocale: en-US
+Publisher: LDraw
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml:7]:
```suggestion
Publisher: LDraw.org
```
Must match Control Panel / Apps&Features

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
+PackageLocale: en-US
+Publisher: LDraw
+PackageName: LDraw Parts Library
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml:8]:
```suggestion
PackageName: LDraw
```
To match Control Panel for upgrade matching

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
+Installers:
+- InstallerUrl: https://www.ldraw.org/files/aioi/LDraw_All-In-One-Installer_2023-02_setup_64Bit_v1.exe
+  Architecture: x64
+  InstallerType: exe
+  InstallerSha256: AD2338A58BCAAAE0FB3EE3FC8E87F5502625E134E0BFACB3413ADEE1B9720F93
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.installer.yaml:10]:
```suggestion
  InstallerSha256: AD2338A58BCAAAE0FB3EE3FC8E87F5502625E134E0BFACB3413ADEE1B9720F93
  ProductCode: LDraw (64 Bit) 2023-02 v1
  Scope: machine
```

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.locale.en-US.yaml:5]:
```suggestion
PackageIdentifier: LDraw.AIOI.2023
PackageVersion: 2023-02 v1
```

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.yaml:5]:
```suggestion
PackageIdentifier: LDraw.AIOI.2023
PackageVersion: 2023-02 v1
```

File: manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: LDraw.AIOI
+PackageVersion: 9.5.0.0
```
Comment (mdanish-kh) [inline manifests/l/LDraw/AIOI/9.5.0.0/LDraw.AIOI.installer.yaml:5]:
```suggestion
PackageIdentifier: LDraw.AIOI.2023
PackageVersion: 2023-02 v1
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
You'll also have to modify the directory structure to match the new PackageId and PackageVersion. `manifests/l/LDraw/AIOI/9.5.0.0` -> `manifests/l/LDraw/AIOI/2023/2023-02 v1`. The `.yaml` files would also require re-naming to match the new PackageIdentifier.

**Review by mdanish-kh — COMMENTED** [review]
The `.yaml`'s also need renaming:

`LDraw.AIOI.installer.yaml` -> `LDraw.AIOI.2023.installer.yaml`

Similar renaming required for the other two .yaml files

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**